src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+html,
+body,
+header,
+nav,
+div,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --theme-color: #00acdc;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
  --theme-color: #00acdc;
}
```

Reply:
Seems like you forgot to fix this comment.